### PR TITLE
docs: add uno platform to readme

### PR DESCRIPTION
**Pull request by ghuntley**
**What kind of change does this PR introduce?**

docs

**What is the current behavior?**

undocumented

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Install the following packages to start building your own ReactiveUI app. <b>Not
 | Xamarin.iOS       | [ReactiveUI][IosDoc]                | [![CoreBadge]][Core] | [ReactiveUI.Events][CoreEvents]         |
 | Xamarin.Mac       | [ReactiveUI][MacDoc]                | [![CoreBadge]][Core] | [ReactiveUI.Events][CoreEvents]         |
 | Tizen             | [ReactiveUI][CoreDoc]               | [![CoreBadge]][Core] | [ReactiveUI.Events][CoreEvents]         |
+| Platform Uno      | [ReactiveUI.Uno][UnoDoc]            [ [![CoreBadge]][Uno]  | [Coming soon][UnoEvents]                |
```
Comment (worldbeater) [inline README.md]: Noticed a minor typo:

```suggestion
| Platform Uno      | [ReactiveUI.Uno][UnoDoc]            | [![UnoBadge]][Uno]  | [Coming soon][UnoEvents]                |
```

**Comment by glennawatson** [comment]
Let's not list the Uno events at the moment is my recommendation.

**Comment by glennawatson** [comment]
The event packages are getting deprecated so we shouldn't be adding new ones

**Comment by ghuntley** [comment]
Opened up https://github.com/reactiveui/website/issues/349 to track documentation updates needed when events packages become deprecated.

**Comment by glennawatson** [comment]
Will submit a new pr with the advice submitted tomorrow and clean up the docs. Will add you to the approval list.

**Pull request merged**